src/runtime/Object.h

```diff
@@ -583,6 +583,8 @@ class Object : public PointerValue {
     }
 #endif
 
+    virtual bool isConcatSpreadable(ExecutionState& state);
```

Comment:
Should `isConcatSpreadable` be defined as virtual function? If there is no other overriding function, normal function would be more suitable.